account_journal_security/models/account_journal.py

```diff
@@ -111,6 +111,24 @@ def _search(self, domain, offset=0, limit=None, order=None, access_rights_uid=No
                 '|', ('modification_user_ids', '=', False),
                 ('id', 'not in', user.journal_ids.mapped('id'))]
```

Comment:
Aca no seria mas performante usar ids en vez de mapped?  
`                ('id', 'not in', user.journal_ids.ids)]
`

ya se.... pero ya que estamos, digo